app/src/main/java/at/bitfire/icsdroid/ui/InfoActivity.kt

```diff
@@ -72,4 +83,114 @@ class InfoActivity: AppCompatActivity() {
         }
     }
 
+    class LibsFragment: Fragment() {
+        @Composable
+        fun TextDialog(@StringRes descriptionRes: Int, onDismissRequest: () -> Unit) {
+            AlertDialog(
+                onDismissRequest = onDismissRequest,
+                confirmButton = {
+                    TextButton(onClick = onDismissRequest) {
+                        Text(stringResource(android.R.string.ok))
+                    }
+                },
+                text = {
+                    Text(
+                        text = stringResource(descriptionRes),
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                }
+            )
+        }
+
+        override fun onCreateView(
+            inflater: LayoutInflater,
+            container: ViewGroup?,
+            savedInstanceState: Bundle?
+        ): View = ComposeView(requireContext()).apply {
+            setContent {
+                val context = LocalContext.current
+
+                MdcTheme {
+                    var showingLicenseDialog by remember { mutableStateOf(false) }
+                    if (showingLicenseDialog)
+                        TextDialog(R.string.app_info_gplv3_note) { showingLicenseDialog = false }
+
+                    var showingDonateDialog by remember { mutableStateOf(false) }
+                    if (showingDonateDialog)
+                        TextDialog(R.string.donate_message) { showingDonateDialog = false }
+                    
+                    LibrariesContainer(
+                        modifier = Modifier.fillMaxSize(),
+                        header = {
+                            // Main Header
+                            item {
```

Comment:
Hi @ArnyminerZ , just a remark: 
maybe it's more elegant to create a composable that contains the whole upper part of the screen with its own column instead of putting each element within the LibrariesContainer's LazyColumn in a separate item {}-block :-)